tests/test_asyncio/test_commands.py

```diff
@@ -106,13 +106,13 @@ async def test_command_on_invalid_key_type(self, r: redis.Redis):
     async def test_acl_cat_no_category(self, r: redis.Redis):
         categories = await r.acl_cat()
         assert isinstance(categories, list)
-        assert "read" in categories
+        assert "read" in categories or b"read" in categories
```

Comment:
`
x = ['read', read']
assert any(map(lambda v: v in x, res))`
etc for others. WDYT?

Reply:
as above